boards/imix/src/main.rs

```diff
@@ -249,8 +286,8 @@ pub unsafe fn reset_handler() {
     fx0_i2c.set_client(fx0);
 
     // Clear sensors enable pin to enable sensor rail
-    sam4l::gpio::PC[16].enable_output();
-    sam4l::gpio::PC[16].clear();
+    // sam4l::gpio::PC[16].enable_output();
+    // sam4l::gpio::PC[16].clear();
```

Comment:
Well, if you leave this in, then you can't use the radio with existing (my?) hardware. Upon initializing the SPI, the device fails to boot.